packages/relay/src/routes/commentRoute.ts

```diff
@@ -83,4 +85,30 @@ export default (
                 res.json({ txHash })
             })
         )
+
+    app.get('/api/comment/commentHistory', async (req, res) => {
+        try {
+            const fromEpoch = parseInt(req.query.from_epoch as string)
+            const toEpoch = parseInt(req.query.to_epoch as string)
+
+            if (isNaN(fromEpoch) || isNaN(toEpoch)) {
+                return res
```

Comment:
Should we use `InvalidParametersError` as well?

Reply:
fixed